flink-core/src/main/java/org/apache/flink/api/common/ArchivedExecutionConfig.java

```diff
@@ -68,7 +74,28 @@ public ArchivedExecutionConfig(
             boolean objectReuseEnabled,
             long periodicMaterializeIntervalMillis,
             Map<String, String> globalJobParameters) {
+        this(
+                executionMode,
+                "UNKNOW",
+                restartStrategyDescription,
+                maxParallelism,
+                parallelism,
+                objectReuseEnabled,
+                periodicMaterializeIntervalMillis,
+                globalJobParameters);
+    }
+
+    public ArchivedExecutionConfig(
+            String executionMode,
+            String jobType,
+            String restartStrategyDescription,
+            int maxParallelism,
+            int parallelism,
+            boolean objectReuseEnabled,
+            long periodicMaterializeIntervalMillis,
+            Map<String, String> globalJobParameters) {
```

Comment:
Why do you add a constructor? could we reuse the last constructor and add the `String jobType`?